examples_source/2D_simulation(crystalline)/plot_5_COASTER_Rb2CrO4.py

```diff
@@ -53,7 +58,10 @@
             spectral_width=4e4,  # in Hz
             reference_offset=-8e3,  # in Hz
             label="3Q dimension",
-            events=[SpectralEvent(transition_query=[{"ch1": {"P": [3], "D": [0]}}])],
+            events=[
+                SpectralEvent(transition_query=[{"ch1": {"P": [3], "D": [0]}}]),
+                MixingEvent(query="TotalMixing"),
```

Comment:
This would be a `nomixing` event. I don't know of any example with `totalmixing`.

Reply:
Should be changed now. I think then the only place to show `TotalMixing` is when Lexi is constructing the 3QMAS method in the new writing custom methods section. Again, total mixing is not necessary, but it would be good to include somewhere in the documentation for completeness.